ja-JP-mac/browser/browser/genai.ftl

```diff
@@ -143,6 +160,15 @@ genai-onboarding-select-primary = チャットを始める
 genai-chatbot-contextual-title = タブを切り替えずに AI チャットボットを使用する
 genai-chatbot-contextual-subtitle = { -brand-short-name } のサイドバーに AI チャットボットを追加すると、チャットとブラウザーを並べて表示できます。
 genai-chatbot-contextual-button = チャットボットを選ぶ
+genai-chatbot-summarize-title = 新機能！ ワンクリックでページを要約
+genai-chatbot-summarize-button = ページを要約
+# “Summarize Page” should be consistent with the translation for the string genai-menu-summarize-page
+genai-chatbot-summarize-sidebar-provider-subtitle = サイドバーで AI チャットボットを右クリックし、“{ genai-menu-summarize-page }” を選択します。
```

Comment:
Not sure about the choice to use the message reference, while English uses the text. 

While very unlikely, English might remove `genai-menu-summarize-page` without requiring changes to this group of strings. It would be on Japanese to make sure localization is updated in that case.

```
      genai.ftl
          WARNING: Obsolete message reference: genai-menu-summarize-page at line 166, column 81 for genai-chatbot-summarize-sidebar-provider-subtitle
          WARNING: Obsolete message reference: genai-menu-summarize-page at line 168, column 72 for genai-chatbot-summarize-sidebar-generic-subtitle
          WARNING: Obsolete message reference: genai-menu-summarize-page at line 170, column 80 for genai-chatbot-summarize-footer-provider-subtitle
```

Reply:
Thank you for telling that...
I'll replace them when the string has removed.